pkg/datapath/linux/bigtcp/bigtcp.go

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bigtcp
+
+import (
+	datapathOption "github.com/cilium/cilium/pkg/datapath/option"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/probes"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+const (
+	defaultGROMaxSize int = 65536
+	defaultGSOMaxSize int = 65536
+
+	BIGTCPGROMaxSize int = 196608
+	BIGTCPGSOMaxSize int = 196608
```

Comment:
These do not seem to be used outside the package, so:

```suggestion
	bigTCPGROMaxSize int = 196608
	bigTCPGSOMaxSize int = 196608
```

Reply:
They weren't big for export purposes (although that happens), but because that's the real name of the feature BIG-TCP.
Anyway, I'll make them small to avoid exporting them